backend/src/interface/persistent_data/model.rs

```diff
@@ -70,7 +75,7 @@ pub struct Side {
     pub price: Price,
 }
 
-#[derive(Default)]
+#[derive(Debug, Default, PartialEq, Clone)]
```

Comment:
add Eq aswell

Reply:
Same problem as with Meal: f32 cannot be Eq'ed